### DISCUSSION TITLE
Blank login route

### DISCUSSION THREAD

**Discussion by VirendraArekar** [laravel/framework#34593] [General]
* I created website with laravel 5.8 and it work fine on ubuntu and windows 10  with auth. But when i upload that website on server it work well but only login route not working. It show me blank or nothing. I try to debug it by commenting Auth::routes() and add own login route with callback function but it print nothing and also i try to catch request from LoginController through __construct and also LoginController@showLoginForm function but nothing happen. Actually not call properly.  For solving that problem i fresh install laravel 5.8 and create auth again using php artisan make:auth and and remaining code copy and paste but it wont work on server but on local windows 10 and ubuntu work well. Please help me.

**Comment by rodrigopedra** [comment]
Things to check:

- Do you have an `.env` file in your production server? If so, is the `APP_KEY` variable set?
  - As you said you are copying the project to the server, some OS/FTP clients can treat `.env` file as a hidden file
- Are you copying the `vendor` folder?
  - It is recommended to run `composer install` if you have `SSH` access to your server, but if you can't, for any reason, check if this folder is being copied with your project
- Do `storage` and `bootstrap/cache` folders have write permissions on production server?
  - reference: https://laravel.com/docs/8.x/installation#configuration
  - reference for 5.8: https://laravel.com/docs/5.8/installation#configuration
- If `storage` is writable, did you check any output on `storage/logs/laravel.log` (or similar file if you are using the daily log) for error output?
- Do any other pages work? If all your other routes are password protected, try creating a simple unauthenticated route for testing.

Also, bear in mind this kind of question is better suited to the help forums and not to the issue tracker as stated in documentation:

> Laravel's GitHub issue trackers are not intended to provide Laravel help or support...

reference: https://laravel.com/docs/8.x/contributions#support-questions

The link above lists several Laravel Support and Help forums supported by the community.
